### DISCUSSION TITLE
View mode!

### DISCUSSION THREAD

**Discussion by c7dl8p** [DanielRendox/RoutineTracker#20] [Feature Requests]
plz add an option to change the view from weekly to monthly 

![Screenshot_20240606-183747](https://github.com/DanielRendox/RoutineTracker/assets/112339868/a64a121d-4162-4294-95ee-b878473b2889)

**Comment by DanielRendox** [comment]
Not sure about this design, but thank you for the suggestion. We may consider adding it in the long term.
